docs/guides/chip_tool_guide.md

```diff
@@ -668,6 +674,57 @@ The following flags are available:
     Here, _<onoff\>_ is a `[0/1]` flag, which when set to `1` prints the trace
     data with automation logs to the console.
 
+<hr>
+
+### Commissioner name and ID flags
+
+All CHIP Tool commands can be used together with the following
+commissioner-related flags:
+
+-   `--commissioner-name`
+-   `--commissioner-nodeid`
+
+These flags let you inform the Matter device about which commissioner it must
+interact with. They are especially useful in scenarios where you are working
+with more than one fabric, but you can also use it with a single commissioner.
+
+#### `--commissioner-name` flag
+
+As per the CHIP Tool implementation, the commissioners are required to have the
+following names: `alpha` for the first one, `beta` for the second one, `gamma`
+for the third one, `4` for the fourth one, `5` for the fifth one, and so on.
+
+If you don't use this flag, the CHIP Tool assumes that the command is meant for
+the `alpha` commissioner.
+
+**Example of commands:**
+
+```
+$ ./chip-tool any subscribe-by-id '0x0028,0x0028,0x0101,0x0028,0x0028,0x0028' '5,6,0,1,2,4' 100 1000 1 '0,0,1,0,0,0' --keepSubscriptions true
+$ ./chip-tool any subscribe-by-id '0x0028,0x0028,0x0101,0x0028,0x0028,0x0028' '5,6,0,1,2,4' 100 1000 2 '0,0,1,0,0,0' --keepSubscriptions true --commissioner-name beta
+$ ./chip-tool any subscribe-by-id '0x0028,0x0028,0x0101,0x0028,0x0028,0x0028' '5,6,0,1,2,4' 100 1000 3 '0,0,1,0,0,0' --keepSubscriptions true --commissioner-name gamma
+```
+
+#### `--commissioner-nodeid` flag
+
+Each commissioner has a node ID that you can use instead of its name when
+providing the commissioning flags.
+
+If you don't use this flag, the CHIP Tool assumes that the command is meant for
+the `alpha` commissioner with the default node ID `112233`.
+
+**Example of commands:**
+
+```
+chip-tool pairing ble-thread 12378411096372871992 hex:0e08000000000001000035060004001fffe00708fd6be52118a6b9000410205c6257d29b2115d3065fdb6e855f830c0402a0f7f8000300000d0102adda02081111111122224589051000aa22dd445566778899aabbccddeeff030a4f70656e546872656164 20202021 3840 --commissioner-name alpha --commissioner-nodeid 123321
+```
+
+```
+any subscribe-by-id '0x0028,0x0028,0x0101,0x0028,0x0028,0x0028' '5,6,0,1,2,4' 100 1000 3 '0,0,1,0,0,0' --keepSubscriptions true --commissioner-name gamma --commissioner-nodeid 123321
```

Comment:
```suggestion
chip-tool basicinformation read vendor-id --commissioner-name alpha --commissioner-nodeid 123321
```
We can add note that if the device has been already commissioned by controller's nodeid e.g. 123321 you need to always add such nodeid param, If you don't provide `--commissioner nodeid`, the default value will be taken (112233) and the communication will fail, unless the ACL on the device will be updated.